Tasks/google-play-release/GooglePlay.ts

```diff
@@ -264,24 +377,17 @@ function updateTrack(edits: any, packageName: string, track: string, versionCode
  * @param apkVersionCodes
  * @returns {*}
  */
-function uploadChangeLogs(edits: any, changelogFile: string, apkVersionCodes: any) {
-    try {
-        let stats: fs.Stats = fs.statSync(changelogFile);
-        if (stats && stats.isFile()) {
-            let apkEdit: Promise<void> = Promise.resolve();
-            apkVersionCodes.forEach(function (apkVersionCode) {
-                apkEdit = apkEdit.then(function (res) {
-                    // console.log("Adding changelog file...");
-                    console.log(tl.loc('AddChangelog'));
-                    return addChangelog(edits, 'en-US', changelogFile, apkVersionCode);
-                });
-            });
-            return apkEdit;
+async function uploadCommonChangeLog(edits: any, languageCode: string, changelogFile: string, apkVersionCodes: number[]) {
+    let stats: fs.Stats = fs.statSync(changelogFile);
+
+    if (stats && stats.isFile()) {
+        for (let apkVersionCode of apkVersionCodes) {
+            tl.debug(`Adding the changelog file ${changelogFile} to the APK version code ${apkVersionCode}`);
+            await addChangelog(edits, 'en-US', changelogFile, apkVersionCode);
+            tl.debug(`Successfully added the changelog file ${changelogFile} to the APK version code ${apkVersionCode}`);
```

Comment:
Debug statements that use "changelog" in the string should use two words ("change log") for consistency.